apis/server_management/scripts/wazuh_server_management_apid.py

```diff
@@ -194,8 +194,8 @@ def start(params: dict, config: ManagementAPIConfig):
             )
         )
 
-    rbac_manager = RBACManager()
     commands_manager = CommandsManager()
+    rbac_manager = RBACManager()
```

Comment:
This is just to make sure the commands manager starts first, because the RBAC manager depends on it. Both should be up by the time we start using them though, it's just to be correct.